content/news/035/index.md

```diff
@@ -71,6 +71,22 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### [Brontefy Me Devlog #3][hedgein-devlog-3]
+
+   ![Brontefy Me Devlog #3](brontefyme.png)
+
+   [@hedgein][hedgein-github] continues the [Brontefy Me][hedgein-devlog-3] series!
+   In this latest devlog, she gives an update on where Brontefy Me will be
+   heading and why it slowed down. During a recent [stream][hedgein-stream], it
```

Comment:
I believe in combination with https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1069#discussion_r908836436 ; this could be simplified with 

```suggestion
   heading and why it slowed down. During a recent [hedgein-stream], it
```

Reply:
Actually when reading more thouroughly the documentation, it changes the title but not the displayed text :( ; please ignore.